src/api/middlewares/check_role.js

```diff
@@ -0,0 +1,49 @@
+
+const User = require('../models/user.model');
+const httpStatus = require('http-status');
+
+/**
+ * Check the role of the current user
+ * @author Jayser Mendez.
+ * @public
+ */
+// eslint-disable-next-line
+const checkRole = (role) => {
```

Comment:
This is the only new thing added basically

Reply:
I can see but wouldn't it be better if you can rebase it on top of master :slightly_smiling_face:

Reply:
I did it :/ but all the commits are there again lol

Reply:
okay As this one would be squashed and merge reset your fork to make future better :smile:

Reply:
I will do that lol. Idk why this happened hahhaah